packages/patrol_cli/lib/src/commands/develop.dart

```diff
@@ -112,6 +112,12 @@ class DevelopCommand extends PatrolCommand {
     final displayLabel = boolArg('label');
     final uninstall = boolArg('uninstall');
 
+    String? iOSInstalledAppsEnvVariable;
+    if (device.targetPlatform == TargetPlatform.iOS) {
+      iOSInstalledAppsEnvVariable =
+          await _iosTestBackend.getInstalledAppsEnvVariable(device.id);
+    }
+
```

Comment:
This will work only on iOS simulator.

Reply:
ios only needs this :)

Reply:
What about physical device?

This invocation: `[xcrun', 'simctl', 'listapps', $deviceId]` works only for the simulator.

Reply:
Though I'm not even sure now if `patrol develop` works on iOS, lol.

Anyway thanks for your comment:)